lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -953,6 +989,14 @@ void MtohRenderOverride::_CreateSceneIndicesChainAfterMergingSceneIndex()
     // selection highlighting.
     wfSi->addExcludedSceneRoot(_ID);
     _lastFilteringSceneIndexBeforeCustomFiltering  = wfSi;
+    
+    const unsigned int currentDisplayStyle = drawContext.getDisplayStyle();
+    
+    //Are we using Bounding Box display style ?
+    if (currentDisplayStyle & MHWRender::MFrameContext::kBoundingBox){
+        //Insert the bounding box filtering scene index which converts geometries into a bounding box using the extent attribute
+        _lastFilteringSceneIndexBeforeCustomFiltering = Fvp::BboxSceneIndex::New(_lastFilteringSceneIndexBeforeCustomFiltering);
```

Comment:
Couldn't this always be in the chain and simply be a pass-through if bounding box mode isn't on?

Reply:
Please see my comment above.